app/MonorepoRoutes.jsx

```diff
@@ -20,9 +20,9 @@ function MonorepoRoutes() {
 MonorepoRoutes.createRouteFromReactElement = (element, parentRoute) => {
   const monorepoRoutes = createRoutesFromReactChildren(
     <Route path='projects'>
-      {SLUGS.map(slug => <MonorepoRoute path={slug} />)}
+      {SLUGS.map(slug => <MonorepoRoute key={slug} path={slug} />)}
       <Route path=':locale'>
-        {SLUGS.map(slug => <MonorepoRoute path={slug} />)}
+        {SLUGS.map(slug => <MonorepoRoute key={slug} path={slug} />)}
```

Comment:
Not really related to the other changes here, but the missing key warnings made real React errors harder to spot while I was chasing down a couple of bugs in this PR. So this makes the console a little bit less noisy.

Reply:
#7116 also go in the way of debugging, so I might fix that here too.